scripts/core/editor3/components/Editor3Component.tsx

```diff
@@ -52,6 +52,7 @@ const EVENT_TYPES_TRIGGER_DROP_ZONE = [
     ...MEDIA_TYPES_TRIGGER_DROP_ZONE,
     EDITOR_BLOCK_TYPE,
     'Files',
+    'application/superdesk.compatible.embed',
```

Comment:
would be nice to describe how this works and link to the line that handles dropped embeds